Assets/HoloToolkit/Input/Tests/Scripts/DisplaySpeechKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class DisplaySpeechKeywords : MonoBehaviour
+    {
+        public SpeechInputSource speechInputSource;
+        public Text textPanel;
```

Comment:
This is based on the existing [DisplayKeywords.cs](https://github.com/Microsoft/HoloToolkit-Unity/blob/0397cfab806d3e56dd18dffa4b90f4b511493f97/Assets/HoloToolkit/Input/Tests/Scripts/DisplayKeywords.cs)